src/SDKs/CognitiveServices/dataPlane/Vision/Face/Face/Microsoft.Azure.CognitiveServices.Vision.Face.csproj

```diff
@@ -6,14 +6,15 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Vision.Face</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services Face APIs.</Description>
-    <Version>2.1.0-preview</Version>
+    <Version>3.0.0-preview</Version>
```

Comment:
Is there a reason why this is a major version bump?

Reply:
Please create a properties.cs similar to [this](https://github.com/lebronJ/azure-sdk-for-net/blob/0e3f092cf127d84eeacc49325604cd7fa5703bef/src/SDKs/CognitiveServices/dataPlane/Search/BingCustomSearch/BingCustomSearch/Properties/AssemblyInfo.cs) and assign the appropriate version numbers

Reply:
By adding support of LargeFaceList and LargePersonGroup, there are some breaking changes in those operations like FindSimilar, Identify and Verify. So we propose a major version upgrade.

Reply:
When in preview, you can release a breaking change with a minor version bump, i.e., `2.2.0-preview`

Reply:
OK, we will follow that rule. Thanks for the information.

Reply:
Resolved by adding `properties.cs` and change version to `2.2.0-preview`. Regenerate SDK by master swagger and regenerate test records. Thanks.